source/codegen/common_helpers.py

```diff
@@ -27,7 +27,7 @@ def get_input_and_output_custom_types(functions):
   return (input_custom_types, output_custom_types)
 
 def is_string_arg(parameter):
-  return parameter['type'] in ['ViChar[]', 'ViInt8[]', 'ViUInt8[]']
+  return parameter['type'] in ['ViChar[]', 'ViInt8[]', 'ViUInt8[]', 'char[]', 'uInt8[]']
```

Comment:
Could this be based on the `grpc_type`? I'd like to use hardcoded driver-specific types less if possible. Keying on `grpc_type` can be a cheap way to do that, though we may need to build up something else (not in this change).